### PR TITLE
IR-5143 GroundPlane Component and ColorInput issues fix

### PR DESCRIPTION
**Pull request by SamMazerIR**
## Summary
extra padding in groundPlane node editor, removing a comment in image/SourceProperties, fixing hyperactive reactivity in color picker

## References
[https://tsu.atlassian.net/browse/IR-5143](https://tsu.atlassian.net/browse/IR-5143)

## QA Steps
open/close the color picker in groundPlane component, change colors, click away, should see expected behavior of a color picker

check/uncheck the visibility of the ground plane, should not only work, but show expected behavior

### CODE REVIEW

File: packages/ui/src/primitives/tailwind/Color/index.tsx [hunk 1]
```diff
@@ -52,36 +52,68 @@ export function ColorInput({
...
+        !inputRef.current.contains(event.target) &&
+        pickerRef.current &&
+        !pickerRef.current.contains(event.target)
+      ) {
+        setIsPickerOpen(false)
+        if (onRelease) onRelease(color)
+      }
+    }
+
+    document.addEventListener('mousedown', handleClickOutside)
```
Comment (SYBIOTE) [inline packages/ui/src/primitives/tailwind/Color/index.tsx:83]: can we use the client input system to handle this ?

Comment (SamMazerIR) [reply packages/ui/src/primitives/tailwind/Color/index.tsx:83]: I don't think so, no

**Review by DanielBelmes — CHANGES_REQUESTED** [review]
Looks like this one was supposed to be based off release0.9 but isn't.

**Comment by DanielBelmes** [comment]
Reopening for 0.9 base

**Pull request closed without merging**